packages/browser/examples/demoClientApp/src/DemoClientApp.js

```diff
@@ -132,6 +66,7 @@ class DemoClientApp extends Component {
 
     this.state = {
       status: "loading",
+      // FIXME: This is currently wrong on session restore, as we don't expose the Issuer from ISessionInfo
```

Comment:
@NSeydoux this is where I hit into the issue with not having the issuer exposed.